README.md

```diff
@@ -688,3 +688,85 @@ We recommend querying using payment key hashes (`addr_vkh`) when possible (other
   }
   ```
 </details>
+<details>
+  <summary>/oracles/getDatapoints</summary>
+  This endpoint is used to return specific data (data point) of a specified oracle.
+
+  There are two usages of this endpoint - with and without source.
+  This is because when calling with source, ticker becomes mandatory.
+
+  1. without specifying source:
+
+  Input
+
+  ```js
+  {
+    addresses: Array<string>, // mandatory, bech32 addresses of trusted oracles
+    ticker?: string,  // optional. If not set, fetch all available tickers. If set, fetch only that ticker
+    block?: number, // optional. If not set, fetch latest `count` data and order desc. If set, find `count` nearest values around that block
```

Comment:
Also, update the comment for `block` please to specify that it should be block **number** (ideally, I'd also change the parameter name itself, block is confusing as I'd expect it to be a block hash for rollback protection) and also clarify better how it will affect the selection, `nearest values around that block` is too vague, from the query it seems it will always select **AFTER OR WITHIN** that block number, comment should specify that, otherwise the contract is not clear and a braking change is possible. Ideally I'd rename the parameter into `sinceBlocknum`

Reply:
Changed + added more verbose description stating it's **ABSOLUTE** distance. It's a weird concept, but if you think about it, it could be useful to get nearest data **around** a block (closest data in time you could get), not just later than, since you can't really know when the data could have been posted.